web/gds-user-ui/src/components/Section/NetworkActivity/NetworkActivity.tsx

```diff
@@ -26,28 +27,39 @@ const sharedAxisStyles = {
 };
 
 const NetworkActivity = () => {
+ /* const { data } = useFetchNetworkActivity();
```

Comment:
This is commented out for now to continue displaying the mock data while the network activity endpoint returns a 501.